sowm.c

```diff
@@ -30,6 +30,19 @@ static void (*events[LASTEvent])(XEvent *e) = {
 
 #include "config.h"
 
+void win_move(const Arg arg) {
+    int  r = arg.com[0][0] == 'r';
+    char m = arg.com[1][0];
+
+    win_size(cur->w, &wx, &wy, &ww, &wh);
+
+    XMoveResizeWindow(d, cur->w, \
+        wx + (r ? 0 : m == 'e' ?  arg.i : m == 'w' ? -arg.i : 0),
+        wy + (r ? 0 : m == 'n' ? -arg.i : m == 's' ?  arg.i : 0),
+        ww + (r ?     m == 'e' ?  arg.i : m == 'w' ? -arg.i : 0 : 0),
+        wh + (r ?     m == 'n' ? -arg.i : m == 's' ?  arg.i : 0 : 0));
```

Comment:
Just need to wrap these two in `MAX(1, code_here)`.

Reply:
Will do